react/src/lib/components/Menu/components/Icon/Icon.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import PropTypes from "prop-types";
+import * as edsIcons from "@equinor/eds-icons";
+import { IconData } from "@equinor/eds-icons";
+import { Icon as EdsIcon } from "@equinor/eds-core-react";
+
+type IconProps = {
+    icon: string;
+    active?: boolean;
+    className?: string;
+};
+
+export const Icon: React.FC<IconProps> = (props) => {
```

Comment:
If it only was a wrapper for `eds-icon`, I'd agree. However, the long-term plan is to also include other icon libraries and maybe also our own icons (if required at some point, e.g. the vector selector icons). That is why I'd prefer to continue calling it `Icon` already in order to prevent any later refactoring.